view/theme/frio/templates/admin/users.tpl

```diff
@@ -0,0 +1,272 @@
+<script>
+	function confirm_delete(uname){
+		return confirm( "{{$confirm_delete}}".format(uname));
+	}
+	function confirm_delete_multi(){
+		return confirm("{{$confirm_delete_multi}}");
+	}
+	function selectall(cls){
+		$("."+cls).prop("checked", true);
+		return false;
+	}
+	function unselectall(cls){
+		$("."+cls).prop("checked", false);
+		return false;
+	}
+	function details(uid) {
+		$("#user-"+uid+"-detail").toggleClass("hidden");
+		return false;
+	}
+</script>
+<style>
+	tr.details td,
+	tr.details th
+		{ border-top: 0!important; }
+</style>
+
+<div class="panel panel-default">
```

Comment:
please use an all-embracing div for the page content.
```
<div id="admin-users" class="adminpage  generic-page-wrapper">
   html content
</div>
```

The classes `adminpage`  `generic-page-wrapper` are already styled through the `style.css`

Reply:
yes, but everything will be smaller, because the page wrapper has padding.

Reply:
I don't think we need a wrapper around the panels as @fabrixxm laid them out.

Reply:
I would like to have a global wrapper to give js and css the possibility to exactly address specific elements.
+ Having a background color for the page - so it is consistent with the other page.
You can add an own class and overwrite existing css (the solution I would prefer for consistency reasons - e.g. change page stylings of commen pages through changing the style info of one class) or create an own class and remove the the classes you don't wan't